transcom/mymove
# B 18448 deprecate personally procured moves fields

B 18448

## Summary

Removed PersonallyProcuredMoves code from ShipmentSummaryWorksheet in order to be able to remove PersonallyProcuredMoves from Move object. Due to extensive modifications needed to ShipmentSummaryWorksheet, this component will need to be tested thoroughly.

##Testing Steps
1. Verify all related Unit tests pass (Changes were all Go, but could indirectly impact JS).
2. As a Customer, Create a PPM under a new move.
3. Approve it as a Service Counselor
4. When signing the shipment as Customer verify that all fields are being populated, and no errors occur
